bitbots_docs/docs/manual/tutorials/launch_files.rst

```diff
@@ -1,219 +1,40 @@
-+------------------+------------+-------------------------------------------------------------------------------------+
-|use_game_settings | false      | siehe Teamplayer Launchskript                                                       |
-+------------------+------------+-------------------------------------------------------------------------------------+
-
-- simple_behavior.launch
-
-Startet das simple Behavior. Nur zum Ball laufen und dann den Ball kicken.
-
-- Head Behavior
-
-- head_behavior.launch
-
-Startet das Head Behavior
-
-+------------------+------------+-------------------------------------------------------------------------------------+
-|Parameter         |Defaultwert |  Erklärung                                                                          |
-+------------------+------------+-------------------------------------------------------------------------------------+
-|depends_only      | false      | startet nur die dependency, für debugging Zwecke in Pycharm (deprecated)            |
-+------------------+------------+-------------------------------------------------------------------------------------+
-|use_game_settings | false      | siehe Teamplayer Launchskript                                                       |
-+------------------+------------+-------------------------------------------------------------------------------------+
-
-head_behavior_standalone.launch
-
-+------------------+------------+-------------------------------------------------------------------------------------+
-|Parameter         |Defaultwert |  Erklärung                                                                          |
-+------------------+------------+-------------------------------------------------------------------------------------+
-|depends_only      | false      | startet nur die dependency, für debugging Zwecke in Pycharm (deprecated)            |
-+------------------+------------+-------------------------------------------------------------------------------------+
-|wolfgang          | true       | Definiert ob das Head Behavior für die Wolfgang Plattform gestartet werden soll     |
-+------------------+------------+-------------------------------------------------------------------------------------+
-
-Lowlevel
-________
-- Buttons
-- Ros Control
-
-Misc
-____
-- Teleop
-- System Monitor
-- Bringup
-
-Die übergreifenden Launchskripte aus diesem Kapitel sind bereits oben erklärt. Hier werden noch die restlichen Launchfiles beschrieben.
-
-Motion
-_______
-- Animation Server
-- HCM
-- Walking
-
-Navigation
-__________
-- Localization
-- Pathfinding
-
-Vision
-______
-- Imageloader
-- Vision
-
-Base Footprint
-______________
-
-Humanoid League Misc
-____________________
-- Game Controller
-- Speaker
-- Team Communication
-- Transformer
-
-Humanoid League Visualisation
-_____________________________
-- Interactive Marker
-
-UDP Bridge
```

Comment:
I would readd this, because it will actually be needed for the RoboCup

Reply:
Why is it needed? Is there not a package documentation for the UDP Bridge?

Reply:
It might I'm not sure. But the UDP Bridge also needs to be started locally during a game which is a special case.
So when I would look through the docs of what can and needs to be launched 'regularly', this is probably where I would expect to find this